docs/DG_Diagrams/ModelComponent.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+
```

Comment:
![e3](https://user-images.githubusercontent.com/60377900/97545507-176bd180-1a06-11eb-935c-e757c1af9483.png)

Since the box is empty with just the class name, maybe you can consider removing the empty box in between and just leave the class name by itself to make it neater and increase readability.